src/html.js

```diff
@@ -67,6 +71,16 @@ export default class Html extends PureComponent {
 
           <link href="/asset/favicon.png" rel="shortcut icon" />
           <link rel="stylesheet" href="/asset/normalize.css" />
+          <link rel="stylesheet" href="/asset/normalize.css" />
```

Comment:
mostly looking good, only this duplicate `<link rel="stylesheet" href="/asset/normalize.css" />`, is it copied by accident?

Reply:
Oops, it was an accident...
Updated in https://github.com/twreporter/twreporter-react/pull/2216/commits/d32a02e6f38620564a481711b5425423d60fc72f.